service/borrowerService.js

```diff
@@ -0,0 +1,73 @@
+let dao = require("../dao/borrowerDAO.js");
+let moment = require("moment");
```

Comment:
const would be more fitting than let for these two, but of course it works either way.